### PR TITLE
DOC-10406 Clarify debug zip --timeout flag as "per request"

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-10406

In cockroach-debug-zip.md, clarified that --timeout flag is "per request".

Rendered preview: [cockroach debug zip General flags](https://deploy-preview-18613--cockroachdb-docs.netlify.app/docs/v24.1/cockroach-debug-zip.html#general)

### CODE REVIEW

**Review by xinhaoz — APPROVED** [review]

File: src/current/v24.1/cockroach-debug-zip.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Flag | Description
...
 `--redact` | Redact sensitive data from the generated `.zip`, with the exception of range keys, which must remain unredacted because they are essential to support CockroachDB. This flag replaces the deprecated `--redact-logs` flag, which only applied to log messages contained within `.zip`. See [Redact sensitive information](#redact-sensitive-information) for an example.
 `--redact-logs` | **Deprecated** Redact sensitive data from collected log files only. Use the `--redact` flag instead, which redacts sensitive data across the entire generated `.zip` as well as the collected log files. Passing the `--redact-logs` flag will be interpreted as the `--redact` flag.
-`--timeout` | Return an error if the command does not conclude within a specified nonzero value. The timeout is suffixed with `s` (seconds), `m` (minutes), or `h` (hours). For example:<br /><br />`--timeout=2m`
+`--timeout` | Return an error if the command does not conclude within a specified nonzero value. The timeout is suffixed with `s` (seconds), `m` (minutes), or `h` (hours). In the process of generating a debug zip, many requests are made. Each request is subject to this timeout limit.<br /><br />**Default:** `60s`
```
Comment (taroface) [inline src/current/v24.1/cockroach-debug-zip.md:121]: Optional suggestions for readability (not sure if "internal" is correct):

```suggestion
`--timeout` | Return an error if the `debug zip` command does not conclude within a specified nonzero value. The timeout is suffixed with `s` (seconds), `m` (minutes), or `h` (hours).<br /><br />In the process of generating a debug zip, many internal requests are made. Each request is subject to this timeout limit.<br /><br />**Default:** `60s`
```

**Review by taroface — APPROVED** [review]
LGTM, a few suggested tweaks

File: src/current/v24.1/cockroach-debug-zip.md (hunk 1 above)
Comment (florence-crl) [reply src/current/v24.1/cockroach-debug-zip.md]: discussed with @taroface and updated.

File: src/current/v24.1/cockroach-debug-zip.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ Flag | Description
...
 `--redact-logs` | **Deprecated** Redact sensitive data from collected log files only. Use the `--redact` flag instead, which redacts sensitive data across the entire generated `.zip` as well as the collected log files. Passing the `--redact-logs` flag will be interpreted as the `--redact` flag.
-`--timeout` | Return an error if the command does not conclude within a specified nonzero value. The timeout is suffixed with `s` (seconds), `m` (minutes), or `h` (hours). For example:<br /><br />`--timeout=2m`
+`--timeout` | In the process of generating a debug zip, many internal requests are made. Each request is subject to this timeout limit.<br /><br />Return an error if an internal request does not conclude within a specified nonzero value. The timeout is suffixed with `s` (seconds), `m` (minutes), or `h` (hours).<br /><br />**Default:** `60s`
```
Comment (taroface) [inline src/current/v24.1/cockroach-debug-zip.md:121]: This seems more straightforward to me, but I'm not sure how much better it is. I'll defer to you.

```suggestion
`--timeout` | Return an error if any of the multiple requests used to generate a debug zip does not conclude within a specified nonzero value. The timeout is suffixed with `s` (seconds), `m` (minutes), or `h` (hours).<br /><br />**Default:** `60s`
```

**Pull request merged**